Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-
-# Specify your gem's dependencies in tictactoe-ruby.gemspec
-gemspec
+gem "rspec"
+gem "rake"
+gem "bundler"
```

Comment:
Presumably if you have Gemfile that means you have already installed bundler.

Reply:
OOh, thanks for letting me know.